mk/iso.mk

```diff
@@ -14,18 +14,23 @@ $(BUILD)/iso_casper.tag: $(BUILD)/live $(BUILD)/chroot.tag $(BUILD)/live.tag $(B
 	# Create new casper directory
 	mkdir -p "$(BUILD)/iso/$(CASPER_PATH)"
 
+ifeq ($(DISTRO_MACHINE),x13s)
+	# copy over dtb to casper
+	cp -v "$(BUILD)/live/usr/lib/linux-image-6.9.1+/qcom/sc8280xp-lenovo-thinkpad-x13s.dtb" "$(BUILD)/iso/$(CASPER_PATH)/$(DISTRO_MACHINE_DTB)"
```

Comment:
Why not copy _all_ DTBs? That way, more ARM platforms that boot using DT over ACPI. I know that NixOS and Rocky Linux currently boot like this: U-Boot -> GRUB -> Linux. While you really only need the FDT in U-Boot and can boot without the in-kernel DTB, it's not recommended.

Jeremy and Aron: Sorry for hijacking this :P

Reply:
Hmm how do you select the right one when its available, you need to do so before loading linux.

Reply:
Oh look @travmurav seems to be on it https://github.com/TravMurav/dtbloader

Reply:
You just place all DTBs in `/boot/dtb` and the bootloader picks it up.

Reply:
> You just place all DTBs in `/boot/dtb` and the bootloader picks it up.

... which one? OTOH, as described, initramfs is a bit special (won't boot, or only from nvme, if the modules and firmware hooks aren't there). For me this sounds a lot like you still need a device specific layer of code, even if we manage to get something like dtbloader figures out the right dtb. But this would be doable for one image, at least.

Reply:
Putting the DTBs in the necessary paths is _already_ handled by the .deb package of the kernel. This shouldn't be done manually.

Reply:
... if it works. It depends on flash-kernel, and my impression was that it doesn't, haven't found the cause yet. Therefore the direct command as a workaround / hack, as the whole thing currently is.